<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminForm</class>
 <widget class="QWidget" name="AdminForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员控制台</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabCompany">
      <attribute name="title">
       <string>公司信息</string>
      </attribute>
      <layout class="QVBoxLayout" name="companyVerticalLayout">
       <item>
        <widget class="QLabel" name="labelCompanyName">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot;font-size:14pt;&quot;&gt;所属公司：加载中...（ID：-1）&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refreshFlight">
         <property name="text">
          <string>刷 新</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableFlight">
         <property name="editTriggers">
          <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPlane">
      <attribute name="title">
       <string>飞机管理</string>
      </attribute>
      <layout class="QVBoxLayout" name="planeVerticalLayout">
       <item>
        <layout class="QHBoxLayout" name="planeOpHorizontalLayout">
         <item>
          <widget class="QLabel" name="labelPlaneType">
           <property name="text">
            <string>选择机型：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbxPlaneType"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddPlane">
           <property name="text">
            <string>添加飞机</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeletePlane">
           <property name="text">
            <string>删除选中</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tablePlane">
         <property name="editTriggers">
          <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRecord">
      <attribute name="title">
       <string>操作日志</string>
      </attribute>
      <layout class="QVBoxLayout" name="recordVerticalLayout">
       <item>
        <widget class="QTableView" name="tableRecords">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
